technologydata/datasources/specification/sources.schema.json

```diff
@@ -60,6 +60,24 @@
                 "required": false
             }
         },
+        {
+            "name": "url_date",
+            "type": "string",
+            "description": "The timestamp of the archiving on Internet Archive Wayback Machine.",
+            "example": "2020-08-07 12:12:01",
```

Comment:
@euronion add more info . 

* when it was accessed / parsed / and archived.

Reply:
@finozzifa Looking at this, Maybe it makes sense to split this into `url_access_date` and `url_archive_date` for archiving and access?
* `url_archive_date` will get populated automatically, because it is also used for automatic retrieval. For this we can keep the description like "The timestamp of the archived version of the original document on the Internet Archive Wayback Machine; this should be as close to or identical to the url_access_date of when the sources has been accessed."
* `url_access_date` would need to be filled out manually and can be (but shouldn't be) empty. It would represent something like "The day and time the original document was accessed and data extracted."